src/test/java/de/tum/in/www1/artemis/ManagementResourceIntegrationTest.java

```diff
@@ -0,0 +1,123 @@
+package de.tum.in.www1.artemis;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.verify;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Optional;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.SpyBean;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.test.context.support.WithMockUser;
+
+import de.tum.in.www1.artemis.config.audit.AuditEventConverter;
+import de.tum.in.www1.artemis.domain.PersistentAuditEvent;
+import de.tum.in.www1.artemis.service.AuditEventService;
+import de.tum.in.www1.artemis.service.feature.Feature;
+import de.tum.in.www1.artemis.util.DatabaseUtilService;
+import de.tum.in.www1.artemis.util.RequestUtilService;
+
+public class ManagementResourceIntegrationTest extends AbstractSpringIntegrationTest {
+
+    @Autowired
+    DatabaseUtilService database;
+
+    @Autowired
+    RequestUtilService request;
+
+    @SpyBean
+    AuditEventService auditEventService;
+
+    @Autowired
+    AuditEventConverter auditEventConverter;
+
+    private List<PersistentAuditEvent> persAuditEvents;
+
+    @BeforeEach
+    public void initTestCase() {
+        database.addUsers(2, 0, 0);
+        var persAuditEvent = new PersistentAuditEvent();
+        persAuditEvent.setPrincipal("student1");
+        persAuditEvent.setAuditEventDate(Instant.now());
+        persAuditEvent.setAuditEventType("type");
+        var data = new HashMap<String, String>();
+        data.put("1", "2");
+        persAuditEvent.setId(1L);
+        persAuditEvent.setData(data);
+
+        var persAuditEvent2 = new PersistentAuditEvent();
+        persAuditEvent2.setPrincipal("student2");
+        persAuditEvent2.setAuditEventDate(Instant.now().minus(5, ChronoUnit.DAYS));
+        persAuditEvent2.setAuditEventType("tt");
+        persAuditEvent2.setData(data);
+        persAuditEvents = new ArrayList<>() {
```

Comment:
I tried to store them first in the database, but this didn't work for GitHub CI tests. It broke a lot of other tests. I guess it is because database stored a lot of other types of audit events. So I use a Spy in this case and mock the calls to the database